certified-connectors/PPM Express/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2934 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PPM Express",
+    "version": "v1.0",
+    "description": "PPM Express is an Online Project Portfolio Management Software. This connector allows you to retrieve information about the entities(Project, Idea, Task, Key Date, Resource) in your tenant, as well as to create new or update existing ones.",

```

Comment:
@vsorokin-ppmexpress please make sure that everything is grammatically correct and is free of typographical errors.
The whitespace is missing here `... entities(Project ...`.